MAUI/Calendar/events.md

```diff
@@ -0,0 +1,182 @@
+---
+layout: post
+title: Events in .NET MAUI Calendar control | Syncfusion
+description: Learn here all about the Events support in Syncfusion .NET MAUI Calendar (SfCalendar) control and more details.
+platform: maui
+control: SfCalendar
+documentation: ug
+---
+
+# Events in .NET MAUI Calendar (SfCalendar)
+
+The Calendar has the events `ViewChanged`, `SelectionChanged`, `Tapped`, `DoubleTapped` and `LongPressed` for notifying after user interactions in [.NET MAUI Calendar]. 
+
+## ViewChanged
+
+The [ViewChanged] event is used to notify when the current view of calendar is changed while the view is swiped to previous or next view, and when the calendar view is switched to another calendar view.
```

Comment:
The [ViewChanged] event is used to notify when the current view of the calendar is changed while the view is swiped to the previous or next view, as well as when the calendar view is switched to another calendar view.

Reply:
Content changed